bigquery/tables.rb

```diff
@@ -253,11 +355,22 @@ def run_query_as_job project_id:, query_string:
                                          dataset_id:   ARGV.shift,
                                          table_id:     ARGV.shift,
                                          storage_path: ARGV.shift
+  when "import_gcs_json"
```

Comment:
Should it have a command-line option? I left it out because the command line user doesn't define the schema when calling `import_gcs_json` anyway. Having a command line option for autodetect would have the same arguments and outcome as the `import_gcs_json` one.

Reply:
That makes perfect sense. But: Yep! They should all be callable from the CLI
_(even if they're not truly "useful" ~ it's for exercising the sample)_

FYI #misc _Node.js and a number of other languages write their tests by calling the CLI and verifying the output (Ruby just calls the method for now, but it's good to follow the same pattern, & maybe Ruby will call the CLI too)_

Reply:
Good to know! Added the command line option in 3ce0922